python/hsfs/feature_view.py

```diff
@@ -235,11 +290,17 @@ def init_serving(
                 If set to False, the online feature store storage connector is used which relies on the private IP.
                 Defaults to True if connection to Hopsworks is established from external environment (e.g AWS
                 Sagemaker or Google Colab), otherwise to False.
+            init_online_store_sql_client: bool, optional. If set to True, initialise the SQL client to retrieve feature vector(s)
+                from the online feature store. Defaults to True if init_online_store_rest_client is False, otherwise False.
+            init_online_store_rest_client: bool, optional. If set to True, initialise the Online Store REST Client to retrieve
+                feature vector(s) from the online feature store. Defaults to False, meaning the sql client will be initialised.
             options: Additional options as key/value pairs for configuring online serving engine.
                 * key: kwargs of SqlAlchemy engine creation (See: https://docs.sqlalchemy.org/en/20/core/engines.html#sqlalchemy.create_engine).
                   For example: `{"pool_size": 10}`
+                * key: "config_online_store_rest_client" - dict, optional. Optional configuration options to override defaults for the Online Store REST Client.
+                * key: "reset_online_store_rest_client" - bool, optional. If set to True, the Online Store REST Client will be reset. Provide
```

Comment:
I don't understand what's the purpose of this `reset_online_store_rest_client` flag?

Reply:
Force overwriting the connection with new config details